src/Nowruz/modules/general/components/iconDropDown/iconDropDown.module.scss

```diff
@@ -18,7 +18,7 @@
 .menuList {
   display: flex;
   flex-direction: column;
-  position: absolute;
+  position: absolute !important;
```

Comment:
@marjan-ara 
In general, can we avoid using !important? It will be difficult to maintain it in the longrun. 
Let me know if MenuList is imported from a third party library like Material UI, then it might be unavoidable.

Reply:
You are right. We should avoid it as much as possible, but here MenuLust is imported from MUI and I wanted to rewrite this style. I had to use !important.